src/studio/src/designer/frontend/shared/utils/networking.ts

```diff
@@ -38,6 +38,18 @@ export async function put(
   }
 }
 
+export async function del(
```

Comment:
`delete` would be a more descriptive name

Reply:
delete is a reserved keyword

Reply:
You are right of course 😅